jib-core/src/integration-test/java/com/google/cloud/tools/jib/registry/AuthenticationMethodRetrieverIntegrationTest.java

```diff
@@ -24,17 +25,20 @@
 /** Integration tests for {@link AuthenticationMethodRetriever}. */
 public class AuthenticationMethodRetrieverIntegrationTest {
 
+  private static final EmptyBuildLogger BUILD_LOGGER = new EmptyBuildLogger();
```

Comment:
Can't this just be a mock?

Reply:
Wasn't sure if it was a good idea to use mocks in integration tests.